Dockerfile-devshell

```diff
@@ -0,0 +1,45 @@
+# Uwazi Development Container (not for production!)
+
+# https://getfedora.org
+FROM fedora:30
+
+# Initially built by Michael Vorburger.ch at Google for Huridocs, later maintained by the Uwazi open source community
+LABEL maintainer="Michael Vorburger.ch <vorburger@google.com>"
+
+# Update base image (the public one is typically outdated and contains already patched security vulnerabilities)
+RUN dnf update -y
+
+# https://docs.mongodb.com/v3.4/tutorial/install-mongodb-on-red-hat/
+COPY mongodb-org-3.4.repo /etc/yum.repos.d/mongodb-org-3.4.repo
+
+# Install utility packages generally useful in any devshell
+RUN dnf install -y git findutils less which \
+
```

Comment:
Just a quick comment, maybe this empty line could cause problems, but on this case was just one warning if using docker-compose. Maybe just remove it?

```
# fititnt at bravo in /alligo/code/vorburger/uwazi on git:devshell o [22:37:01]
$ docker-compose --version
docker-compose version 1.21.2, build a133471

# (...)

[WARNING]: Empty continuation line found in:
    RUN dnf install -y git findutils less which     libpng12-devel libpng-devel mongodb-org-shell mongodb-org-tools poppler poppler-utils
[WARNING]: Empty continuation lines will become errors in a future release
```

![Captura de tela de 2019-10-11 22-35-49](https://user-images.githubusercontent.com/812299/66692718-7e8ddb00-ec77-11e9-9c59-5a9d64a17cab.png)